Makefile.in

```diff
@@ -34,10 +34,10 @@ else
 MAYBE_DISABLE_VERIFY=
 endif
 
-ifdef CFG_ENABLE_OPTIMIZE
-OPT_FLAG=--release
-else
+ifdef CFG_DISABLE_OPTIMIZE
```

Comment:
Hm these seem like they may be unrelated changes, perhaps a rebase is necessary?